src/elements/content-sidebar/skills/keywords/EditableKeywords.js

```diff
@@ -154,7 +154,7 @@ class EditableKeywords extends React.PureComponent<Props, State> {
         const { onSave, onCancel }: Props = this.props;
         const { pills, keyword }: State = this.state;
         return (
-            <span className="pill-selector-wrapper">
+            <span className="bdl-PillSelectorDropdown">
```

Comment:
what do you think about using a different classname, like `bdl-EditableKeywords`? since `bdl-PillSelectorDropdown` is used for the PillSelectorDropdown component?